### PR TITLE
[FIX] Fixed pricelist problem

### PR DESCRIPTION
**Pull request by cialuo**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/xie8899/project/pull/41%23discussion_r57557822%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202321921%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23discussion_r57558845%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202327762%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202332225%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23discussion_r57666679%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202684382%22%2C%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202688494%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/41%23issuecomment-202321921%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Others%2C%20LGTM%22%2C%20%22created_at%22%3A%20%222016-03-28T09%3A36%3A30Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40elicoidal%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222016-03-28T09%3A59%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-28T10%3A09%3A58Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7600613%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/elicoidal%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20%40elicoidal%20%22%2C%20%22created_at%22%3A%20%222016-03-29T03%3A08%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7767792%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cialuo%22%7D%7D%2C%20%7B%22body%22%3A%20%22LGTM%22%2C%20%22created_at%22%3A%20%222016-03-29T03%3A26%3A10Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2037c67db434bc67257b81b679e0a9a4077c07433c%20business_requirement_deliverable_default/models/business_requirement_deliverable_default.py%209%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/xie8899/project/pull/41%23discussion_r57557822%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%60%60%60python%5Cr%5Cnif%20self.product_id%5Cr%5Cn%60%60%60%22%2C%20%22created_at%22%3A%20%222016-03-28T09%3A36%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40duanyp1991%20there%20has%20used%20product%20variable%20instead%20self.product_id%22%2C%20%22created_at%22%3A%20%222016-03-28T09%3A55%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7767792%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/cialuo%22%7D%7D%2C%20%7B%22body%22%3A%20%22OK%22%2C%20%22created_at%22%3A%20%222016-03-29T03%3A04%3A28Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/3785252%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/duanyp1991%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20business_requirement_deliverable_default/models/business_requirement_deliverable_default.py%3AL15-25%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 37c67db434bc67257b81b679e0a9a4077c07433c business_requirement_deliverable_default/models/business_requirement_deliverable_default.py 9'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/41#discussion_r57557822'>File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py:L15-25</a></b>
- <a href='https://github.com/duanyp1991'></a> 

``` python
if self.product_id
```
- <a href='https://github.com/cialuo'></a> @duanyp1991 there has used product variable instead self.product_id
- <a href='https://github.com/duanyp1991'></a> OK
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/xie8899/project/pull/41#issuecomment-202321921'>General Comment</a></b>
- <a href='https://github.com/duanyp1991'></a> Others, LGTM
- <a href='https://github.com/duanyp1991'></a> @elicoidal
- <a href='https://github.com/elicoidal'></a> LGTM
- <a href='https://github.com/cialuo'></a> @duanyp1991 @elicoidal
- <a href='https://github.com/duanyp1991'></a> LGTM

<a href='https://www.codereviewhub.com/xie8899/project/pull/41?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/xie8899/project/pull/41?approve=1'></a>&nbsp;<a href='https://github.com/xie8899/project/pull/41'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by xie8899** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/xie8899/project/pull/41'>Refresh</a>.<a href='#crh-update-none'></a>

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py [hunk 1]
```diff
@@ -15,18 +15,10 @@ def _prepare_resource_lines(self):
     @api.multi
     @api.onchange('product_id')
     def product_id_change(self):
-        description = ''
-        uom_id = False
-        unit_price = 0
+        super(BusinessRequirementDeliverable, self).product_id_change()
         product = self.product_id
         if product:
```
Comment (duanyp1991) [inline business_requirement_deliverable_default/models/business_requirement_deliverable_default.py:20]:
``` python
if self.product_id
```

**Comment by duanyp1991** [comment]
Others, LGTM

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py (hunk 1 above)
Comment (cialuo) [inline business_requirement_deliverable_default/models/business_requirement_deliverable_default.py:20]: @duanyp1991 there has used product variable instead self.product_id

**Comment by duanyp1991** [comment]
@elicoidal

**Comment by elicoidal** [comment]
LGTM

File: business_requirement_deliverable_default/models/business_requirement_deliverable_default.py (hunk 1 above)
Comment (duanyp1991) [inline business_requirement_deliverable_default/models/business_requirement_deliverable_default.py:20]: OK

**Comment by cialuo** [comment]
@duanyp1991 @elicoidal

**Comment by duanyp1991** [comment]
LGTM

**Pull request merged**